packages/web-components/src/components/copy/copy.ts

```diff
@@ -92,7 +92,10 @@ class CDSCopy extends CDSIconButton {
 
   connectedCallback() {
     this.closeOnActivation = false;
-    this.align = 'bottom';
+
+    this.align = this.align || 'bottom';
+
+    this.autoalign = this.autoalign || false;
```

Comment:
I'm fairly certain these aren't needed anymore, since they're already have default values in `copy-button`. You can test by removing these lines, and updating the value in Storybook controls should still work.